services/QuillLMS/client/app/bundles/PremiumHub/containers/diagnosticGrowthReports/overviewSection.tsx

```diff
@@ -127,7 +125,7 @@ export const OverviewSection = ({
 
   React.useEffect(() => {
     // this is for testing purposes; this value will always be null in a non-testing environment
-    if (!passedData) {
+    if (!passedData && groupByValue) {
```

Comment:
Why don't we want to call `getData` if there's a `groupByValue`?

Reply:
I'm not sure I follow... do you mean why don't we want to call `getData` if there's **no** `groupByValue`? We need this value for the `aggregateOverviewData` function which is called after `getData`